### PR TITLE
Suggestion for Outline style

### PR DESCRIPTION
**Pull request by YuliKl**
Modified colors for Outline style based on @mdtauk's [proposal](https://github.com/microsoft/microsoft-ui-xaml-specs/pull/39#issuecomment-516553295).

### CODE REVIEW

File: active/ButtonStyles/ButtonStyles.md [hunk 1]
```diff
@@ -56,18 +56,18 @@ Button is composed of the following UI elements:
...
+| Hover |1. #FF000000 (100% Black) *no change* <br/>2. #1A000000 (10% Black) |1. #FFFFFFFF (100% White) *no change* <br/>2. 100% AccentLight1 color |1. #FF000000 (100% Black) <br/> 2. #1A000000 (10% Black) <br/>4. #66000000 (40% Black) |1. #FF000000 (100% Black) <br/>2. #1A000000 (10% Black) |
+| Pressed |1. #FF000000 (100% Black) *no change* <br/>2. #66000000 (40% Black) *no change* |1. #FFFFFFFF (100% White) *no change* <br/>2. 100% AccentDark1 color |1. #FF000000 (100% Black) <br/>2. #33000000 (20% Black) <br/>4. #66000000 (40% Black) |1. #FF000000 (100% Black) <br/>2. #33000000 (20% Black) |
+| Disabled |1. #66000000 (40% Black) *no change* <br/>2. #33000000 (20% Black) *no change* |1. #66000000 (40% Black) *no change* <br/>2. #33000000 (20% Black) *no change*  |1. #66000000 (40% Black) <br/>2. #1A000000 (10% Black) <br/> 4. Transparent color  |1. #66000000 (40% Black) <br/>2. Transparent color |
```
Comment (chigy) [inline active/ButtonStyles/ButtonStyles.md]:
```suggestion
| Rest |1. #FFFFFFFF (100% White) *no change* <br/>2. 100% Accent color *no change* |1. #FF000000 (100% Black) *no change* <br/>2. #33000000 (20% Black) *no change* |1. #FF000000 (100% Black) <br/>2.#FFFFFFFF (100% White) <br/>4. #66000000 (40% Black) |1. #FF000000 (100% Black) <br/>2. Transparent color |
| Hover |1. #FFFFFFFF (100% White) *no change* <br/>2. 100% AccentLight1 color|1. #FF000000 (100% Black) *no change* <br/>2. #1A000000 (10% Black)  |1. #FF000000 (100% Black) <br/> 2. #1A000000 (10% Black) <br/>4. #66000000 (40% Black) |1. #FF000000 (100% Black) <br/>2. #1A000000 (10% Black) |
| Pressed |1. #FFFFFFFF (100% White) *no change* <br/>2. 100% AccentDark1 color|1. #FF000000 (100% Black) *no change* <br/>2. #66000000 (40% Black) *no change*  |1. #FF000000 (100% Black) <br/>2. #33000000 (20% Black) <br/>4. #66000000 (40% Black) |1. #FF000000 (100% Black) <br/>2. #33000000 (20% Black) |
| Disabled |1. #66000000 (40% Black) *no change* <br/>2. #33000000 (20% Black) *no change* |1. #66000000 (40% Black) *no change* <br/>2. #33000000 (20% Black) *no change*  |1. #66000000 (40% Black) <br/>2. #1A000000 (10% Black) <br/> 4. Transparent color  |1. #66000000 (40% Black) <br/>2. Transparent color |
```

I swapped Fill and Accent style in another update and this one is conflicting. Fixing.

File: active/ButtonStyles/ButtonStyles.md [hunk 2]
```diff
@@ -56,18 +56,18 @@ Button is composed of the following UI elements:
...
-| Hover |1. #FF000000 (100% Black) *no change* <br/>2. #1A000000 (10% Black) |1. #FFFFFFFF (100% White) *no change* <br/>2. 100% AccentLight1 color |1. #FF201F1E <br/> 2. #FFF3F2F1 <br/>4. #FF8A8886 |1. #FF000000 (100% Black) <br/>2. #1A000000 (10% Black) |
-| Pressed |1. #FF000000 (100% Black) *no change* <br/>2. #66000000 (40% Black) *no change* |1. #FFFFFFFF (100% White) *no change* <br/>2. 100% AccentDark1 color |1. #FF201F1E <br/>2. #FFEDEBE9 <br/>4. #FF8A8886 |1. #FF000000 (100% Black) <br/>2. #33000000 (20% Black) |
-| Disabled |1. #66000000 (40% Black) *no change* <br/>2. #33000000 (20% Black) *no change* |1. #66000000 (40% Black) *no change* <br/>2. #33000000 (20% Black) *no change*  |1. #FFA19F9D <br/>2. #FFF3F2F1 <br/> 4. Transparent color  |1. #66000000 (40% Black) <br/>2. Transparent color |
+| Rest |1. #FF000000 (100% Black) *no change* <br/>2. #33000000 (20% Black) *no change* |1. #FFFFFFFF (100% White) *no change* <br/>2. 100% Accent color *no change* |1. #FF000000 (100% Black) <br/>2.#FFFFFFFF (100% White) <br/>4. #66000000 (40% Black) |1. #FF000000 (100% Black) <br/>2. Transparent color |
```
Comment (chigy) [inline active/ButtonStyles/ButtonStyles.md]:
```suggestion
| Rest |1. #FFFFFFFF (100% White) *no change* <br/>2. 100% Accent color *no change* |1. #FF000000 (100% Black) *no change* <br/>2. #33000000 (20% Black) *no change* |1. #FF000000 (100% Black) <br/>2.#FFFFFFFF (100% White) <br/>4. #66000000 (40% Black) |1. #FF000000 (100% Black) <br/>2. Transparent color |
```

File: active/ButtonStyles/ButtonStyles.md [hunk 3]
```diff
@@ -56,18 +56,18 @@ Button is composed of the following UI elements:
...
+| Hover |1. #FF000000 (100% Black) *no change* <br/>2. #1A000000 (10% Black) |1. #FFFFFFFF (100% White) *no change* <br/>2. 100% AccentLight1 color |1. #FF000000 (100% Black) <br/> 2. #1A000000 (10% Black) <br/>4. #66000000 (40% Black) |1. #FF000000 (100% Black) <br/>2. #1A000000 (10% Black) |
```
Comment (chigy) [inline active/ButtonStyles/ButtonStyles.md]:
```suggestion
| Hover |1. #FFFFFFFF (100% White) *no change* <br/>2. 100% AccentLight1 color|1. #FF000000 (100% Black) *no change* <br/>2. #1A000000 (10% Black)  |1. #FF000000 (100% Black) <br/> 2. #1A000000 (10% Black) <br/>4. #66000000 (40% Black) |1. #FF000000 (100% Black) <br/>2. #1A000000 (10% Black) |
```

File: active/ButtonStyles/ButtonStyles.md [hunk 4]
```diff
@@ -56,18 +56,18 @@ Button is composed of the following UI elements:
...
+| Pressed |1. #FF000000 (100% Black) *no change* <br/>2. #66000000 (40% Black) *no change* |1. #FFFFFFFF (100% White) *no change* <br/>2. 100% AccentDark1 color |1. #FF000000 (100% Black) <br/>2. #33000000 (20% Black) <br/>4. #66000000 (40% Black) |1. #FF000000 (100% Black) <br/>2. #33000000 (20% Black) |
```
Comment (chigy) [inline active/ButtonStyles/ButtonStyles.md]:
```suggestion
| Pressed |1. #FFFFFFFF (100% White) *no change* <br/>2. 100% AccentDark1 color|1. #FF000000 (100% Black) *no change* <br/>2. #66000000 (40% Black) *no change*  |1. #FF000000 (100% Black) <br/>2. #33000000 (20% Black) <br/>4. #66000000 (40% Black) |1. #FF000000 (100% Black) <br/>2. #33000000 (20% Black) |
```

File: active/ButtonStyles/ButtonStyles.md [hunk 5]
```diff
@@ -56,18 +56,18 @@ Button is composed of the following UI elements:
...
+| Disabled |1. #66000000 (40% Black) *no change* <br/>2. #33000000 (20% Black) *no change* |1. #66000000 (40% Black) *no change* <br/>2. #33000000 (20% Black) *no change*  |1. #66000000 (40% Black) <br/>2. #1A000000 (10% Black) <br/> 4. Transparent color  |1. #66000000 (40% Black) <br/>2. Transparent color |
```
Comment (chigy) [inline active/ButtonStyles/ButtonStyles.md]:
```suggestion
| Disabled |1. #66000000 (40% Black) *no change* <br/>2. #33000000 (20% Black) *no change* |1. #66000000 (40% Black) *no change* <br/>2. #33000000 (20% Black) *no change*  |1. #66000000 (40% Black) <br/>2. #1A000000 (10% Black) <br/> 4. Transparent color  |1. #66000000 (40% Black) <br/>2. Transparent color |
```

File: active/ButtonStyles/ButtonStyles.md (hunk 2 above)
Comment (chigy) [reply active/ButtonStyles/ButtonStyles.md]: I swapped Accent and Fill based on feedback and I created conflict. This should fix it.

File: active/ButtonStyles/ButtonStyles.md [hunk 6]
```diff
@@ -56,18 +56,18 @@ Button is composed of the following UI elements:
...
-| Rest |1. #FFFFFFFF (100% White) *no change* <br/>2. #33FFFFFF (20% White) *no change* |1. #FFFFFFFF (100% White) *no change* <br/>2. 100% Accent color *no change* |  |1. #FFFFFFFF (100% White) <br/>2. Transparent color |
-| Hover |1. #FFFFFFFF (100% White) *no change* <br/>2. #1AFFFFFF (10% White) |1. #FFFFFFFF (100% White) *no change* <br/>2. 100% AccentDark1 color |  |1. #FFFFFFFF (100% White) <br/>2. #1AFFFFFF (10% White) |
-| Pressed |1. #FFFFFFFF (100% White) *no change* <br/> 2. #66FFFFFF (60% White) *no change* |1. #FFFFFFFF (100% White) *no change* <br/>2. 100% AccentLight1 color |  |1. #FFFFFFFF (100% White) <br/>2. #33FFFFFF (20% White) |
-| Disabled |1. #66FFFFFF (40% White) *no change* <br/>2. #33FFFFFF (20% White) *no change* |1. #66FFFFFF (40% White) *no change* <br/>2. #33FFFFFF (20% White) *no change* |  |1. #66FFFFFF (40% White) <br/>2. Transparent color |
+| Rest |1. #FFFFFFFF (100% White) *no change* <br/>2. #33FFFFFF (20% White) *no change* |1. #FFFFFFFF (100% White) *no change* <br/>2. 100% Accent color *no change* |1. #FFFFFFFF (100% White) <br/>2. #FF000000 (100% Black) <br/>4. #66FFFFFF (40% White) |1. #FFFFFFFF (100% White) <br/>2. Transparent color |
```
Comment (chigy) [inline active/ButtonStyles/ButtonStyles.md]:
```suggestion
| Rest |1. #FFFFFFFF (100% White) *no change* <br/>2. 100% Accent color *no change*|1. #FFFFFFFF (100% White) *no change* <br/>2. #33FFFFFF (20% White) *no change*  |1. #FFFFFFFF (100% White) <br/>2. #FF000000 (100% Black) <br/>4. #66FFFFFF (40% White) |1. #FFFFFFFF (100% White) <br/>2. Transparent color |
```

File: active/ButtonStyles/ButtonStyles.md [hunk 7]
```diff
@@ -56,18 +56,18 @@ Button is composed of the following UI elements:
...
+| Hover |1. #FFFFFFFF (100% White) *no change* <br/>2. #1AFFFFFF (10% White) |1. #FFFFFFFF (100% White) *no change* <br/>2. 100% AccentDark1 color |1. #FFFFFFFF (100% White) <br/>2. #1AFFFFFF (10% White) <br/>4. #66FFFFFF (40% White)|1. #FFFFFFFF (100% White) <br/>2. #1AFFFFFF (10% White) |
```
Comment (chigy) [inline active/ButtonStyles/ButtonStyles.md]:
```suggestion
| Hover |1. #FFFFFFFF (100% White) *no change* <br/>2. 100% AccentDark1 color |1. #FFFFFFFF (100% White) *no change* <br/>2. #1AFFFFFF (10% White) |1. #FFFFFFFF (100% White) <br/>2. #1AFFFFFF (10% White) <br/>4. #66FFFFFF (40% White)|1. #FFFFFFFF (100% White) <br/>2. #1AFFFFFF (10% White) |
```

File: active/ButtonStyles/ButtonStyles.md [hunk 8]
```diff
@@ -56,18 +56,18 @@ Button is composed of the following UI elements:
...
+| Pressed |1. #FFFFFFFF (100% White) *no change* <br/> 2. #66FFFFFF (60% White) *no change* |1. #FFFFFFFF (100% White) *no change* <br/>2. 100% AccentLight1 color |1. #FFFFFFFF (100% White) <br/>2. #33FFFFFF (20% White) <br/>4. #66FFFFFF (40% White) |1. #FFFFFFFF (100% White) <br/>2. #33FFFFFF (20% White) |
```
Comment (chigy) [inline active/ButtonStyles/ButtonStyles.md]:
```suggestion
| Pressed |1. #FFFFFFFF (100% White) *no change* <br/>2. 100% AccentLight1 color |1. #FFFFFFFF (100% White) *no change* <br/> 2. #66FFFFFF (60% White) *no change* |1. #FFFFFFFF (100% White) <br/>2. #33FFFFFF (20% White) <br/>4. #66FFFFFF (40% White) |1. #FFFFFFFF (100% White) <br/>2. #33FFFFFF (20% White) |
```

File: active/ButtonStyles/ButtonStyles.md [hunk 9]
```diff
@@ -56,18 +56,18 @@ Button is composed of the following UI elements:
...
+| Disabled |1. #66FFFFFF (40% White) *no change* <br/>2. #33FFFFFF (20% White) *no change* |1. #66FFFFFF (40% White) *no change* <br/>2. #33FFFFFF (20% White) *no change* |1. #66FFFFFF (40% White) <br/>2. #1AFFFFFF (10% White) <br/> 4. Transparent color |1. #66FFFFFF (40% White) <br/>2. Transparent color |
```
Comment (chigy) [inline active/ButtonStyles/ButtonStyles.md]:
```suggestion
| Disabled |1. #66FFFFFF (40% White) *no change* <br/>2. #33FFFFFF (20% White) *no change* |1. #66FFFFFF (40% White) *no change* <br/>2. #33FFFFFF (20% White) *no change* |1. #66FFFFFF (40% White) <br/>2. #1AFFFFFF (10% White) <br/> 4. Transparent color |1. #66FFFFFF (40% White) <br/>2. Transparent color |
```

**Comment by YuliKl** [comment]
Instead of merging conflicts, I'll just start over

**Pull request closed without merging**